contracts/TopShot.cdc

```diff
@@ -844,7 +844,8 @@ pub contract TopShot: NonFungibleToken {
                     let traitDictionary: {String: AnyStruct} = {
                         "SeriesNumber": TopShot.getSetSeries(setID: self.data.setID),
                         "SetName": TopShot.getSetName(setID: self.data.setID),
-                        "SerialNumber": self.data.serialNumber
+                        "SerialNumber": self.data.serialNumber,
+                        "Locked": TopShotLocking.isLocked(nftRef: &self as! &NonFungibleToken.NFT)
```

Comment:
not sure if you want LockExpiry as well `TopShotLocking.getLockExpiry`

Reply:
Hadn't thought of that! I could go either way. The only way that LockExpiry is useful is if you can mark it as a `Date` and allow users to range on it. As of now, that isn't something Flowty can do